gluoncv/utils/parallel.py

```diff
@@ -93,12 +93,13 @@ def __init__(self, module, ctx_list=None, sync=False):
     def __call__(self, inputs, *targets, **kwargs):
         # the inputs should be the outputs of DataParallelModel
         if not self.ctx_list:
-            return self.module(inputs, *targets, **kwargs)
+            return self.module(*inputs, *targets, **kwargs)
```

Comment:
*(inputs + targets)

Reply:
But when use aux=True in FCN, the input is a tuple. So how to solve it?

Reply:
Thanks! I see the problem. A simple solution is
```
if isinstance(input, NDArray):
    return self.module(inputs, *targets, **kwargs)
else:
    return self.module(*(inputs + targets), **kwargs)
```